src/idaes_compatibility/robustness/unit_models/test_rgibbs.py

```diff
@@ -0,0 +1,714 @@
+            "Vap", "NH3"
+        ]
+    ] = 99999.99999999997
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_in[0.0].mole_frac_phase_comp[
+            "Vap", "O2"
+        ]
+    ] = 5.750431282346176
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].enth_mol_phase["Vap"]
+    ] = 2.5797225634987406e-05
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].enth_mol_phase_comp[
+            "Vap", "CH4"
+        ]
+    ] = 9.131356578608373e-05
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].enth_mol_phase_comp[
+            "Vap", "CO2"
+        ]
+    ] = 3.8432408761344524e-06
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].enth_mol_phase_comp[
+            "Vap", "CO"
+        ]
+    ] = 8.617482264231038e-06
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].enth_mol_phase_comp[
+            "Vap", "H2O"
+        ]
+    ] = 5.614692038136825e-06
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].enth_mol_phase_comp[
+            "Vap", "H2"
+        ]
+    ] = 1.1286872760356127e-05
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].enth_mol_phase_comp[
+            "Vap", "N2"
+        ]
+    ] = 0.0007529656318755848
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].enth_mol_phase_comp[
+            "Vap", "NH3"
+        ]
+    ] = 0.000147611121632822
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].enth_mol_phase_comp[
+            "Vap", "O2"
+        ]
+    ] = 0.00011456899776931075
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].entr_mol_phase_comp[
+            "Vap", "CH4"
+        ]
+    ] = 0.002273113949091129
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].entr_mol_phase_comp[
+            "Vap", "CO2"
+        ]
+    ] = 0.0032746835561418483
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].entr_mol_phase_comp[
+            "Vap", "CO"
+        ]
+    ] = 0.004593745058364088
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].entr_mol_phase_comp[
+            "Vap", "H2O"
+        ]
+    ] = 0.004278478461249582
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].entr_mol_phase_comp[
+            "Vap", "H2"
+        ]
+    ] = 0.005287458822573278
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].entr_mol_phase_comp[
+            "Vap", "N2"
+        ]
+    ] = 0.005016168207707185
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].entr_mol_phase_comp[
+            "Vap", "NH3"
+        ]
+    ] = 0.0029937456289480776
+    m.scaling_factor[
+        m.fs.unit.control_volume.properties_out[0.0].entr_mol_phase_comp[
+            "Vap", "O2"
+        ]
+    ] = 0.003654125560818891
```

Comment:
Entropy is probably used in calculating the Gibbs energy. Note that this scaling was generated using the prototype autoscaling tools (scaling variables by magnitude and constraints by 2-norm). This will eventually be replaced by something better, but will need to wait until the refactor of the scaling tools is completed.

Also note that scaling is not handled my any individual sub-model, but is rather done on the flowsheet as a whole (thus scaling factors for properties and unit model variables all get done at the same time).